config/default-amo.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
   },
   enableTrailingSlashesMiddleware: true,
   fxaConfig: 'amo',
+
+  // This needs to be kept in sync with addons-server's SUPPORTED_NONLOCALES
+  // settings value: https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L274
+  validRootUrlExceptions: [
```

Comment:
Omg yeah and the way we mock out the tests it still passes. Maybe I changed the keys after I tested it in the browser and messed up. /me fixes...